src/services/token.js

```diff
@@ -116,16 +116,23 @@ export class Service {
     const secret = this.options.secret;
     const options = {
       algorithm,
-      expiresIn,
       notBefore,
       audience,
-      issuer,
       jwtid,
-      subject,
       noTimestamp,
       header
     };
 
+    if (!data.iss) {
```

Comment:
Confused a bit about this one @marshallswain. Maybe I'm missing something. Any reason you want to pass those as `data`?

I was thinking any token options should come from `params.jwt` and due to the object deconstruction above this, those fields will just be `undefined` if they weren't passed along via `params.jwt` or aren't a default value.